docker-entrypoint.sh

```diff
@@ -167,6 +167,7 @@ print "Rewriting custom server fragments on /etc/nginx/conf.d/custom/*.conf"
 for filename in /etc/nginx/conf.d/custom/*.conf; do
   if [ -e "${filename}" ] ; then
     cp "${filename}" "${filename}.tmp"
+    sed -e '/#hstsheader/r /templates/hsts.conf' -i "$filename.tmp";
```

Comment:
if we add this, we should add also the other ones.

Reply:
I added what you asked for.